test/integration/forms/action.js

```diff
@@ -305,7 +308,22 @@ context('Patient Action Form', function() {
         return fx;
       })
       .routeFormByAction(_.identity, '11111')
-      .routeLatestFormResponse()
+      .routeLatestFormResponse(() => {
+        return {
+          data: {
+            id: '1',
+            attributes: {
+              status: 'draft',
+              created_at: testTsSubtract(1),
```

Comment:
This adds a "cloud draft" older that localstorage.  if the `this.latestResponse = null;` is removed, this test fails and shows `bazinga` in the foo field